config/routes.yaml

```diff
@@ -1,2 +1 @@
-logout:
-  path: /logout
+
```

Comment:
Is die file dan nog nodig? Aangezien die leeg is?

Reply:
fixed 9de4c285fa85150937ff44a6dda196a4c3e6d126